backend/src/main/kotlin/app/ehrenamtskarte/backend/EntryPoint.kt

```diff
@@ -29,11 +32,35 @@ class Entry : CliktCommand() {
     private val postgresPassword by option()
     private val geocoding by option().choice("true", "false").convert { it.toBoolean() }
     private val geocodingHost by option()
+    private val smtpUsernameEakBayern by option()
+    private val smtpPasswordEakBayern by option()
+    private val smtpUsernameNuernbergPass by option()
+    private val smtpPasswordNuernbergPass by option()
+    private val smtpUsernameShowcase by option()
+    private val smtpPasswordShowcase by option()
 
     override fun run() {
         val backendConfiguration = BackendConfiguration.load(config)
 
         currentContext.obj = backendConfiguration.copy(
+            projects = backendConfiguration.projects.map { project ->
+                project.copy(
+                    smtp = project.smtp.copy(
+                        username = when (project.id) {
+                            EAK_BAYERN_PROJECT -> smtpUsernameEakBayern
+                            NUERNBERG_PASS_PROJECT -> smtpUsernameNuernbergPass
+                            SHOWCASE_PROJECT -> smtpUsernameShowcase
+                            else -> null
+                        } ?: project.smtp.username,
+                        password = when (project.id) {
+                            EAK_BAYERN_PROJECT -> smtpPasswordEakBayern
+                            NUERNBERG_PASS_PROJECT -> smtpPasswordNuernbergPass
+                            SHOWCASE_PROJECT -> smtpPasswordShowcase
+                            else -> null
+                        } ?: project.smtp.password
+                    )
+                )
+            },
```

Comment:
While the CLI config is definitely nice, we won't need it in production use.
I'm not sure if we really need this feature.

Reply:
I can remove it again, if we don't need it.